doc/source/index.rst

```diff
@@ -46,11 +46,17 @@ to implement custom failure criteria and computation.
 
         Provides developer installation and usage information.
 
+The minimum set of inputs to run a postprocessing for composites is the result files of an
+MAPDL solution and a material file (MATML) which was generated by Engineering Data.
+Lay-up files from ACP are optional and only required for some advanced operations. Refer to
+:ref:`Limitations`.
+
 Key features
 ''''''''''''
 
 Here are some key features of PyDPF Composites:
 
+* Postprocessing of layered shell and solid elements, whether they were preprocessed by ACP or not.
```

Comment:
As a user, without context this sentence can be confusing I think (What does it me if the model has not been preprocessed by ACP?).
Postprocessing of layered shell and solid elements. MAPDL models as well as models preprocessed with ACP are supported.